.github/workflows/server-ci.yml

```diff
@@ -69,46 +65,32 @@ jobs:
         run: if [[ -n $(git status --porcelain) ]]; then echo "Please tidy up the Go modules using make modules-tidy"; git diff; exit 1; fi
   golangci:
     name: golangci-lint
+    needs: go
     runs-on: ubuntu-22.04
+    container: mattermostdevelopment/mattermost-build-server:${{ needs.go.outputs.version }}
     defaults:
       run:
         working-directory: server
+    env:
+      GOFLAGS: -buildvcs=false # TODO: work around "error obtaining VCS status: exit status 128" in a container
```

Comment:
This was present in Antonis/Agniva's branch, and seems specific to moving inside a container. I've created https://mattermost.atlassian.net/browse/MM-65181 to revisit this.